pkg/config/config.go

```diff
@@ -10,6 +13,24 @@ func envDefault(key, defaultValue string) string {
 	return value
 }
 
+func envDefaultInt(key, defaultValue string) (int, error) {
+	return strconv.Atoi(envDefault(key, defaultValue))
+}
+
+func envDefaultBool(key string, defaultValue bool) bool {
+	value := os.Getenv(key)
+	if value == "" {
+		return defaultValue
+	}
+
+	for _, v := range []string{"true", "True", "TRUE"} {
```

Comment:
This is super nitpicky and not at all security-related, but is there a specific reason for filtering only these three 'true' strings instead of using 'strings.ToUpper(key)'? I would be the first person to set the environment variable to 'tRUE' because of not hitting the caps-lock key soon enough.

Reply:
Great catch. I've updated accordingly.